instrumentation/active_model_serializers/lib/opentelemetry/instrumentation/active_model_serializers/instrumentation.rb

```diff
@@ -19,13 +19,14 @@ class Instrumentation < OpenTelemetry::Instrumentation::Base
         present do
           !defined?(::ActiveSupport::Notifications).nil? &&
             !defined?(::ActiveModel::Serializers).nil? &&
+            !defined?(::ActiveModelSerializers).nil? &&
```

Comment:
This is the error that comes up on a rails app without `active_model_serializers`
```
WARN -- : Instrumentation: OpenTelemetry::Instrumentation::ActiveModelSerializers unhandled exception during install undefined method `>=' for nil:NilClass
```

Reply:
Hey @johnnyshields does this check look right?  I haven't spent any time with this gem before.

Reply:
@robertlaurin yep this change is correct

Reply:
you can actually remove `!defined?(::ActiveModel::Serializers).nil?` if you want, it's redundant.

Reply:
Will do, thanks for looking it over.